README.md

```diff
@@ -1,35 +1,46 @@
 # Morning challenge - Build the Request module
-This week in your project you will be making API calls from the backend. In the past few weeks you have been making API calls from the frontend with the XMLHttpRequest object and it's methods. Now that you are in the backend we would like you to use the Request module which is [available on npm](https://www.npmjs.com/package/request). This workshop is intended to familiarise you with how this module works by asking you to build a simple version of it.
+This week in your project you will be making API calls from the backend. In the past few weeks you have been making API calls from the frontend with the XMLHttpRequest object and methods. Now that you are in the backend we would like you to learn how to make similar request using the Node core [http module](https://nodejs.org/api/http.html). This workshop is intended to familiarise you with how backend requests are made, and practice using callbacks.
 
 ### Time Required: 1 hour
```

Comment:
Not sure this section is needed. Students should know how a morning challenge goes by Week 5

Reply:
[Removed](https://github.com/foundersandcoders/mc-request-module-workshop/pull/8/commits/922e75e8d598eea525c7484ac5217f0563fa7ca9)